doc/DeveloperGuide.md

```diff
@@ -58,9 +58,12 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
+`* * *` | user | edit an existing person | update details without deleting and adding
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | display a user based on tag | locate people that are marked with a certain tag
 `*` | user with many persons in the address book | sort persons by name | locate a person easily
+`*` | user | get confirmation before deletion | prevent accidental deletion
```

Comment:
This one may go against the "one-shot command" philosophy of Addressbook application